Selenium.Axe/Selenium.Axe/Selenium.Axe.csproj

```diff
@@ -39,4 +40,12 @@
     </EmbeddedResource>
     <EmbeddedResource Include="Resources\scan.js" />
   </ItemGroup>
+  <Choose>
+    <When Condition=" '$(TargetFramework)' != 'netstandard2.0' and '$(TargetFramework)' != 'netstandard2.1'   ">
+      <ItemGroup>
+        <Reference Include="System.Web" />
```

Comment:
This feels a little odd; is this a conditional reference a requirement of `HtmlAgilityPack`?

Reply:
This is related to warning that comes with using of HttpUtility.HtmlEncode.
Here is the warning you get
No way to resolve conflict between "System.Web, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Web". Choosing "System.Web, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" arbitrarily.